vendor/jasmine-only/jasmine-only.js

```diff
@@ -0,0 +1,98 @@
+/* jasmine-only - 0.1.1
+ * Exclusivity spec helpers for jasmine: `describe.only` and `it.only`
+ * https://github.com/davemo/jasmine-only
+ */
+(function() {
```

Comment:
No I mean if we were to submit a pull request for jasmine only, using arrow functions and things like that, it wouldn't work in versions prior to node 4.